spec/jobs/dashboard/provider_data_job_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+module Dashboard
+  RSpec.describe ProviderDataJob do
+    describe '.perform' do
+      let(:provider) { create :provider }
+      subject { described_class.perform_now(provider) }
+
+      let(:geckoboard_client) { double Geckoboard::Client }
+      let(:datasets_client) { double Geckoboard::DatasetsClient }
+      let(:dataset) { double Geckoboard::Dataset }
+
+      before do
+        allow(Geckoboard).to receive(:client).and_return(geckoboard_client)
+        allow(geckoboard_client).to receive(:ping).and_return(true)
+        allow(geckoboard_client).to receive(:datasets).and_return(datasets_client)
+      end
+
+      describe '#perform' do
+        it 'calls the runs the geckoboard feedback updater' do
```

Comment:
Really nitpicky - should this be `it 'calls the geckoboard feedback updater' do`?